t/main.go

```diff
@@ -74,6 +74,8 @@ var (
 		"Clear all the test clusters.")
 	dry = pflag.BoolP("dry", "", false,
 		"Just show how the packages would be executed, without running tests.")
+	noBuild = pflag.BoolP("no-build", "", false,
```

Comment:
Don't use double negatives.
Maybe call it `rebuild-binary` and set it to true by default.